pyquil/api/qpu.py

```diff
@@ -199,6 +190,34 @@ def _run_and_measure_payload(self, quil_program, qubits, trials):
 
         return payload
 
+    def get_job(self, job_id):
```

Comment:
Are `get_job` and `wait_for_job` significantly different between `QPUConnection` and `QVMConnection`?  Is abstracting this this traded off against the "badness" of inheritance?

Reply:
I needed to untrap the error handling and getting/posting so it could be used outside of the connection objects as well (eg. in get_devices)